### ISSUE TITLE
CUDA driver version is insufficient for CUDA runtime version while runnung Tensorflow

### ISSUE THREAD

**Issue by kkjh0723** [NVIDIA/nvidia-docker#399]
My host system is Ubuntu 14.04 with CUDA-8.0, nvidia driver v367.35 and Anaconda2. There are 2x 1080 GPU cards.
I could run Tensorflow tutorial example(mnist-softmax.py) in the host using gpu.
I simply installed TF with pip.

I want to test if I can run same example in the nvidia container.
So I build from `nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04`.
I could run `nvidia-smi`

    +-----------------------------------------------------------------------------+
    | NVIDIA-SMI 367.35                 Driver Version: 367.35                    |
    |-------------------------------+----------------------+----------------------+
    | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
    | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
    |===============================+======================+======================|
    |   0  GeForce GTX 1080    Off  | 0000:01:00.0      On |                  N/A |
    | 34%   35C    P8    10W / 180W |    387MiB /  8113MiB |      7%      Default |
    +-------------------------------+----------------------+----------------------+
    |   1  GeForce GTX 1080    Off  | 0000:02:00.0     Off |                  N/A |
    | 34%   33C    P8    10W / 180W |      1MiB /  8113MiB |      0%      Default |
    +-------------------------------+----------------------+----------------------+
                                                                                   
    +-----------------------------------------------------------------------------+
    | Processes:                                                       GPU Memory |
    |  GPU       PID  Type  Process name                               Usage      |
    |=============================================================================|
    +-----------------------------------------------------------------------------+
Then I installed Anaconda3 and TF and run same example.
But I got following error messages

    Failed to get the number of CUDA devices: CUDA driver version is insufficient for CUDA runtime version
    Floating point exception (core dumped)

Could you please tell me how I can fix this?

**Comment by flx42** [comment]
You should first try with driver version 375. Can you do that?

**Comment by kkjh0723** [comment]
@flx42 Thank for the answer. But according to the prerequisites of `nvidia-docker` ([link](https://github.com/NVIDIA/nvidia-docker/wiki/Installation)), version above 340.29 seems okay isn't it? Can it be a problem?

**Comment by flx42** [comment]
@kkjh0723 look at the line below:
> Your driver version might limit your CUDA capabilities (see [CUDA requirements](https://github.com/NVIDIA/nvidia-docker/wiki/CUDA#requirements))

**Comment by kkjh0723** [comment]
@flx42 Sorry, I missed that part... Now I'm upgrading the driver version to 375 as you recommended.  By the way, do I need to (somhow?) re-install nvidia-docker also?

**Comment by kkjh0723** [comment]
@flx42 after I upgrade the driver, it works like a charm. thanks very much!
